### ISSUE TITLE
GUID: 0bdeb43e-9115-4b00-4fb3-89e5bb55cb47 ERROR: ENOENT: no such file or directory, open 'config/project-scratch-def.json'.

### ISSUE THREAD

**Issue by vektor-knight** [wadewegner/deploy-to-sfdx#46]
When deploying directly to SF

GUID: 0bdeb43e-9115-4b00-4fb3-89e5bb55cb47 ERROR: ENOENT: no such file or directory, open 'config/project-scratch-def.json'.

4) GUID: 0bdeb43e-9115-4b00-4fb3-89e5bb55cb47 ERROR: ENOENT: no such file or directory, open 'config/project-scratch-def.json'.
3) Set instance url to https://uss-serenity-123815.my.salesforce.com.
2) Successfully cloned https://github.com/kevinohara80/sfdc-trigger-framework.
1) No .salesforcedx.yaml found in repository. Using defaults.

**Comment by FabienTaillon** [comment]
To use deploy to sfdx, you need to have a .salesforcedx.yaml file in your repo as explained [here](https://deploy-to-sfdx.com/repo). There is no such file in the repo you're trying to deploy even though the button has been added.

You should create an issue on the [repor](https://github.com/kevinohara80/sfdc-trigger-framework/issues) to ask him to add the file or add it yourself via a pull request.

**Comment by vektor-knight** [comment]
Thanks, figured it was that simple. It was my understanding that his framework may have had a contrived example builtin, which would have triggered on deployment.

**Comment by wadewegner** [comment]
Yes, it does have some built-in capabilities and it will make assumptions. The one missing here is that if it can't find the yml file it will look for a default `config/project-scratch-def.json` in the source. In this case, the repo doesn't appear to have one, so it fails.

This isn't a bad thing to enhance. -- i.e. if it doesn't find one, use a default.
